src/main/groovy/edu/oregonstate/mist/trailsapi/mapper/TrailMapper.groovy

```diff
@@ -3,24 +3,34 @@ package edu.oregonstate.mist.trailsapi.mapper
 import edu.oregonstate.mist.trailsapi.core.Trail
 import org.skife.jdbi.v2.StatementContext
 import org.skife.jdbi.v2.tweak.ResultSetMapper
+import java.sql.Clob
 import java.sql.ResultSet
 import java.sql.SQLException
 
 public class TrailMapper implements ResultSetMapper<Trail> {
-	public Trail map(int i, ResultSet rs, StatementContext sc) throws SQLException {
-		new Trail (
-			id: rs.getInt('ID'),
-			name: rs.getString('Name'),
-			zipCode: rs.getInt('zipCode'),
-			difficulty: rs.getString('DIFFICULTY'),
-			polyline: rs.getString('POLYLINE'),
-			largeDrop: rs.getBoolean('LARGEDROP'),
-			smallDrop: rs.getBoolean('SMALLDROP'),
-			woodRide: rs.getBoolean('WOODRIDE'),
-			skinny: rs.getBoolean('SKINNY'),
-			largeJump: rs.getBoolean('LARGEJUMP'),
-			smallJump: rs.getBoolean('SMALLJUMP'),
-			gap: rs.getBoolean('GAP')
-		)
-	}
+    public Trail map(int i, ResultSet rs, StatementContext sc) throws SQLException {
+    new Trail (
+        id: rs.getInt('ID'),
+        name: rs.getString('NAME'),
+        zipCode: rs.getInt('ZIPCODE'),
+        difficulty: rs.getString('DIFFICULTY_COLOR'),
+        polyline: clobToString(rs.getClob('POLYLINE')),
```

Comment:
Were you able to just use the `rs.getString` method on this clob field? This is what I've done in the past, but I'm not sure if its the right way of doing it. I just assumed `getString` converted the clob to a string.

Reply:
Documentation on `ResultSet.getString()` says nothing about the column data type, only that the "value of the designated column in the current row" is returned as a string (https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSet.html#getString(int)), so it appears to be a valid use.
`rs.getString` works for me as well, so I'll switch to that.